Always add an 'ordering' column to the explorer regardless of permission - fixes #1608

gasman opened the pull request.

kaedroho commented:
Is there ever a time when ``show_ordering_time`` is ``False``? If not, could we remove it?

gasman commented:
Yes - the flag gets checked in _list.html, which is shared between the explore, move and choose views. Ideally _list_explore.html would override the relevant bit of the template so that _list.html doesn't deal with this column at all, but I didn't get round to doing that when I refactored the listings for 1.0.

I've now added `show_ordering_column=0` on the non-explore views, to make this explicit.

gasman commented:
Closed via #1612
